conf/app.routes

```diff
@@ -2,11 +2,11 @@
 
 POST       /traders/:EORI/movements/:movementType                                      uk.gov.hmrc.transitmovements.controllers.MovementsController.createMovement(EORI: EORINumber, movementType: MovementType)
 
-GET        /traders/:EORI/movements/:movementType                                 uk.gov.hmrc.transitmovements.controllers.MovementsController.getMovementsForEori(EORI: EORINumber, movementType: MovementType, updatedSince: Option[OffsetDateTime] ?= None, movementEORI: Option[EORINumber] ?= None, movementReferenceNumber: Option[MovementReferenceNumber] ?= None)
+GET        /traders/:EORI/movements/:movementType                                 uk.gov.hmrc.transitmovements.controllers.MovementsController.getMovementsForEori(EORI: EORINumber, movementType: MovementType, updatedSince: Option[OffsetDateTime] ?= None, movementEORI: Option[EORINumber] ?= None, movementReferenceNumber: Option[MovementReferenceNumber] ?= None, pageNumber: Option[PageNumber] ?= None, itemCount: Option[ItemCount] ?= None, receivedUntil: Option[OffsetDateTime] ?= None)
```

Comment:
Just as said on the `common-transit-convention-traders` PR, the requirements state `page` and `count`. I suppose it matters less for backends, but depending how the parameters are passed through, you may wish to rename them here too.